### DISCUSSION TITLE
Not the best at doing this but... getting an error code during the last step of install using microconda

### DISCUSSION THREAD

**Discussion by BG42424** [neonbjb/tortoise-tts#506] [Q&A]
Basically when I type in the "python setup.py install" step during install I get a runtime error saying that python 3.9 or higher is required. I already have python 3.9 on my CPU BUT I put anaconda on my D drive. Can that be the issue or is there something else I can do to fix this? Would love help :) Thank you for the help :)

p.s. I tried updating the python but it gets a LOT of conflicts so.... again would love anyone willing to help lol When I ask what version I have it is like 3.7 for some reason :P

**Comment by LucidFir** [comment]
Honestly just delete it all and start again imho, I'm no coder though
